pkg/cmd/grafana/apiserver/cmd.go

```diff
@@ -67,16 +78,55 @@ func newCommandStartExampleAPIServer(o *APIServerOptions, stopCh <-chan struct{}
 		factoryOptions.AddFlags(cmd.Flags())
 	}
 
+	o.ExtraOptions.AddFlags(cmd.Flags())
+
 	// Register standard k8s flags with the command line
-	o.RecommendedOptions = options.NewRecommendedOptions(
-		defaultEtcdPathPrefix,
-		grafanaapiserver.Codecs.LegacyCodec(), // the codec is passed to etcd and not used
-	)
 	o.RecommendedOptions.AddFlags(cmd.Flags())
 
 	return cmd
 }
 
+func setupLogging(logLevel int) error {
```

Comment:
With my other comment would suggest moving this function to the pkg/infra/log package to keep the ini file/logic at the same place. We can name it something like `SetupConsoleLogger`. No blocker for this PR though

Reply:
I moved it